db/migrate/20200801170524_add_type_to_user.rb

```diff
@@ -0,0 +1,24 @@
+class User < ApplicationRecord
+  self.table_name = 'users'
+end
+
+class AddTypeToUser < ActiveRecord::Migration[6.0]
+  def up
+    add_column :users, :type, :string
+    add_column :users, :active, :boolean, default: true
+
+    # Set supervisor
+    User.where(role: "supervisor").update_all(type: "Supervisor")
```

Comment:
If this goes in before we "go live" we can probably kill these updates and just reset the database.

Reply:
I would like to keep migrations which demonstrate how to safely migrate even when we have prod data to preserve